recipe/meta.yaml

```diff
@@ -37,18 +37,43 @@ requirements:
     - pytz >=2020.1
 
 test:
+  source_files:
+    - pyproject.toml
   imports:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    #
+    # Also skip `test_rolling_var_numerical_issues` on `ppc64le` as it is a known test failure.
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/149
+    {% set extra_args = ["-c", "pyproject.toml"] %}
```

Comment:
Tried including the `pyproject.toml` file and passing this to pytest to fix the unknown mark issues. Though am seeing a bunch of errors & failures that were not seen before. Am wondering if this is being passed correctly.